controller/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
```

Comment:
please use
testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.7.0'
this lib contains api and engine